### DISCUSSION TITLE
Is a PR to just fix a confusing comment acceptable?

### DISCUSSION THREAD

**Discussion by GalaxiaGuy** [dotnet/runtime#85385] [General] [answered]
The methods

`public static float Min(float val1, float val2)`
`public static double Min(double val1, double val2)`

both include the comment "treats +0 as lesser than -0 as per the specification"

**Comment by huoyaoyuan** [comment]
Why is this confusing? +0 and -0 are different values per IEEE754 specification. You can use `BitConverter.DoubleToUInt64Bits(+0.0)` and `BitConverter.DoubleToUInt64Bits(-0.0)` to confirm.

**Reply by GalaxiaGuy** [reply]
Because the comment appears to be incorrect, based on the implementation and what I understand from the spec, it should say:

"treats -0 as lesser than +0 as per the specification"

**Reply by jkotas** [reply, marked as answer]
You are welcomed to submit PR to correct the comment.

**Reply by huoyaoyuan** [reply]
My fault. `+0 > -0` is correct and institutive.
